actors/common/src/main/java/org/sunbird/learner/util/DataCacheHandler.java

```diff
@@ -33,12 +33,13 @@
 
   @Override
   public void run() {
-    ProjectLogger.log("Data cache started..");
+    ProjectLogger.log("DataCacheHandler:run cache started..", LoggerEnum.INFO.name());
     cache(pageMap, "page_management");
     cache(sectionMap, "page_section");
     roleCache(roleMap);
     orgTypeCache(orgTypeMap);
     cacheSystemConfig(configSettings);
+    ProjectLogger.log("DataCacheHandler:run cache completed..", LoggerEnum.INFO.name());
```

Comment:
From:
"DataCacheHandler:run cache completed.."

To:
"DataCacheHandler:run: Cache refresh completed."